.github/workflows/run_pr_tests.yml

```diff
@@ -163,8 +163,7 @@ jobs:
           debug_support: ON
           install_dir: ${{ github.workspace }}/install_offline
           build_dir: ${{ github.workspace }}/build_offline
-          c_compiler: 'cl.exe'
-          cxx_compiler: 'cl.exe'
+          extra_flags: -DCMAKE_C_COMPILER='cl.exe' -DCMAKE_CXX_COMPILER='cl.exe'
```

Comment:
This is probably the default on windows too.

Reply:
Probably, but also, if it isn't the default, could we use a toolchain file for MSVC?

Reply:
we could but I don't think it's necessary.

Reply:
Windows default. Removed.